packages/lesswrong/lib/routes.ts

```diff
@@ -1775,6 +1775,47 @@ if (isLW) {
   });
 }
 
+addRoute({
+    title: "Think Page",
+    name: 'thinkPage',
+    path: '/think',
+    redirect: () => '/think/posts/new',
+  },
+  {
+    name: 'thinkPagePost',
+    path: '/think/posts/:postId/:slug',
+    componentName: 'ThinkPostPage',
+    background: 'white',
+    title: "Think: Read Post"
+  },
+  {
+    name: 'thinkPageNewPost',
+    path: '/think/posts/new',
+    componentName: 'ThinkPostNew',
+    title: "Think: New Post",
+    background: 'white',
+  },
+  {
+    name: 'thinkPageSequenceOriginal',
+    path: '/think/s/:sequenceId',
```

Comment:
claim: this should be the canonical url format (as noted elsewhere, for consistency)